src/components/Onfido/index.css

```diff
@@ -47,3 +47,7 @@
         height: 92% !important;
     }
 }
+
+.onfido-sdk-ui-NavigationBar-back {
```

Comment:
if you could add the explanation comment why we are hiding this button

Reply:
fixed